README.md

```diff
@@ -70,14 +74,14 @@ Be sure to use version control.
 
 ## `push` a query
 
-**Not Yet Implemented**
+**Implemented!**
 
-`stmocli push [<id>]`
+`stmocli push [<filename>]`
 
-Pushes the current SQL statements and metadata to re:dash for the given query.
-If no query id is specified, push data for all queries.
+Pushes the current SQL statements and metadata to re:dash for the given query file.
+If no filename is specified, push data for all queries.
```

Comment:
It looks like filename is required in the current implementation.

Reply:
I updated the `README` for now, I'll file a follow-up issue for supporting "push all queries"